docs/getting_started/notifications-Examples.md

```diff
@@ -0,0 +1,182 @@
+
+- Configure a small HTTP server that will receive the payload
+- It will then talk to SEKOIA.IO’s Alert API to retrieve more information about the alert itself (its title, its urgency, etc.)
+- The final message is computed and pushed to Microsoft Teams via their incoming WebHook system
+
+Here’s an example of an HTTP server written in Python and FastAPI:
+
+```python title="main.py"
+"""Sample notification handler for SEKOIA.IO WebHook Notifications.
+
+Sample FastAPI application that is in charge of receiving WebHook events from SEKOIA.IO’s Notification mechanism, enrich event by talking to SEKOIA.IO’s Alert API and then to push a notification to Microsoft Teams, via an incoming WebHook.
+
+"""
+
+from datetime import datetime
+from typing import Literal, Optional
+
+import httpx
+from fastapi import BackgroundTasks, FastAPI
+from pydantic import BaseModel, BaseSettings
+
+
+class WebHookSettings(BaseSettings): # (4)
+    apikey: str
+    teams_webhook_url: Optional[str] = None
+
+
+class SEKOIAIOWebHookAlertAttributes(BaseModel):
+    uuid: str
+    short_id: str
+
+
+class SEKOIAIOWebHookMetadata(BaseModel):
+    version: int = 2
+    created_at: datetime
+    community_uuid: str
+    avatar_uuid: Optional[str] = None
+    user_uuid: Optional[str] = None
+    permissions: Optional[list[str]] = None
+
+
+class SEKOIAIOWebHookEvent(BaseModel): # (5)
+    type: Literal["alert"]
+    action: str
+    attributes: SEKOIAIOWebHookAlertAttributes
+    metadata: SEKOIAIOWebHookMetadata
+
+
+app = FastAPI()
+settings = WebHookSettings()
+
+
+async def push_notification_to_teams(event: SEKOIAIOWebHookEvent):
+    """Simple task that will extract the retrieve the full alert via
+    SEKOIA.IO API and then creates a message that will be posted on
+    Microsoft Teams via a predefined incoming WebHook.
+
+    """
+    async with httpx.AsyncClient() as client:
+        response = await client.get( # (2)
+            f"https://api.sekoia.io/v1/sic/alerts/{event.attributes.uuid}",
+            params={"stix": False},
+            headers={"Authorization": f"Bearer {settings.apikey}"},
+        )
+        if response.status_code == 200:
+            alert = response.json()
+            message = (
+                f"A new alert “{alert['title']}” was created on SEKOIA.IO with "
+                "status “{alert['status']['name']}” and “{alert['urgency']['display']}” severity."
+            )
+            if settings.teams_webhook_url:
+                client.post(settings.teams_webhook_url, json={"text": message}) # (3)
+
+
+@app.post("/hooks/67a7472b-b43c-415d-b1ce-c135fb1dfb0e") # (1)
+async def handle_new_alert(
+    event: SEKOIAIOWebHookEvent, background_tasks: BackgroundTasks
+):
+    background_tasks.add_task(push_notification_to_teams, event)
+```
+
+1. Entrypoint of our HTTP server that receives SEKOIA.IO WebHook notifications. Please define a unique URL, using and UUID for example.
```

Comment:
@Men-hau Is this a summary of what the code is doing?

Reply:
these are the steps describe in the original page to make the actions before using the script